Add expiration comment for certificates/credentials in external SSO (#4942)
camunda/camunda-docs: docs/components/console/manage-organization/external-sso.md

We currently support both SAML and Azure Active Directory (Azure AD). As this requires changes in our environment, first raise a ticket in the [support queue](https://jira.camunda.com/projects/SUPPORT/).

:::info

> If you configure an expiring certificate or secret, make sure to record the expiration date; you will need to renew it before that day to avoid a service interruption.

Reviewer: ```suggestion
Expiring certificates or secrets must be renewed prior to their expiration date to avoid a service interruption.
```
